docs/pages/project/contributing/contributing-docs.md

```diff
@@ -401,5 +383,38 @@ The assign tag is used to create a new variable. It is written in the following
 {% capture code_content %}{{ "{% assign variable1 = true " }}%}{% endcapture %}
 {% include code.html code=code_content %}
 
+### Alert
+
+The `alert.html` file is used to format and display notes, warnings, and other informational callouts in Meshery. 
```

Comment:
Add the file link here  - https://github.com/meshery/meshery/blob/master/docs/_includes/alert.html